### DISCUSSION TITLE
[UI] The UI font is questionable

### DISCUSSION THREAD

**Discussion by MerryPanda** [phcode-dev/phoenix#1618] [General]
The main UI font is big and lines blend together because there is not enought space between them vertically (in Project File Tree, eg).

I think that the one used in Brackets originally was quite good.

**Comment by abose** [comment]
Thanks for the feedback @MerryPanda . We have not changed the main ui font settings from brackets. Can you provide a screenshot and point to which region in the UI the fonts/spacing not match brackets?

**Reply by MerryPanda** [reply]
I have 1920x1080 resolution with Windows 125% scale.

The font looks much bigger than it was in Brackets

![image](https://github.com/phcode-dev/phoenix/assets/12458159/c4cc9cdd-8b2b-46bd-8bce-2ae38ac58fe5)

Also the context menu uses large font.

![Untitled](https://github.com/phcode-dev/phoenix/assets/12458159/e7a24529-79ee-4634-9ddc-114a037e6a32)

Something is off in the zooming department, imho.

**Reply by abose** [reply]
The full window zoom feature in phcode is brand new and Brackets doesn't have it. Brackets only has font zoom. The inconsistency should be fixed by July release as a major UI redesign is scheduled for that release.
